idle-infrastructure/fcm/src/main/kotlin/com/swm/idle/infrastructure/fcm/jobposting/service/CreateJobPostingEventService.kt

```
@@ -0,0 +1,52 @@
+class CreateJobPostingEventService(
+    private val fcmClient: FcmClient,
+) {
+
+    private val logger = KotlinLogging.logger {}
+
+    fun send(createJobPostingEvent: CreateJobPostingEvent) {
+        val message = createMessage(createJobPostingEvent)
+
+        try {
+            fcmClient.send(message)
+        } catch (e: Exception) {
```

detekt

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)